tests/models/autoencoder/test_trainer.py

```diff
@@ -98,6 +98,46 @@ def test_trainer_05(self):
         y_test = trainer.predict(model, dataloaders=streamloader, unbatch=True)
         self.assertTupleEqual(self.x_test.shape, y_test.size())
 
+    def test_trainer_06_w_val(self):
+        model = Conv1dAE(seq_len=SEQ_LEN, in_channels=self.x_train.shape[1], enc_channels=(4,))
```

Comment:
Is the sparse AE tested?

Reply:
Yes, see line number 110 in `test_conv.py`